cachelib/allocator/CacheAllocator.h

```diff
@@ -1701,7 +1740,17 @@ class CacheAllocator : public CacheBase {
   // @param fromBgThread use memmove instead of memcopy (for DTO testing)
   // @return true  If the move was completed, and the containers were updated
   //               successfully.
-  bool moveRegularItem(Item& oldItem, WriteHandle& newItemHdl, bool skipAddInMMContainer, bool fromBgThread);
+  bool moveRegularItem(Item& oldItem,
+                       WriteHandle& newItemHdl,
+                       bool skipAddInMMContainer,
+                       bool fromBgThread);
+
+  // Performs book keeping after a regular item is moved to a different memory tier.
+  //
+  // @param oldItem     Reference to the item being moved
+  // @param newItemHdl  Reference to the handle of the new item being moved into
+  // @return true       If the containers were updated successfully.
```

Comment:
Really all that is left is to do, and what this function does is one thing: update in the access container - you could add that to the comments. 

Or change the function name from book keeper to completeAccessContainerUpdate() or something.

Reply:
Changed name of book keeper function to completeAccessContainerUpdate()